unpackaged/vesuvio_calibration/tests/unit/test_calibrate_vesuvio_fit.py

```diff
@@ -660,39 +660,35 @@ def test_fit_peaks_individual(self, group_workspaces_mock):
         group_workspaces_mock.assert_called_once_with(['output_ws_name_Peak_0_Parameters', 'output_ws_name_Peak_1_Parameters'],
                                                       OutputWorkspace='output_ws_name_Peak_Parameters')
 
-    @patch('calibration_scripts.calibrate_vesuvio_fit.EVSCalibrationFit._shared_parameter_fit')
     @patch('calibration_scripts.calibrate_vesuvio_fit.GroupWorkspaces')
-    def test_fit_peaks_shared(self, group_workspaces_mock, shared_parameter_fit_mock):
+    def test_fit_peaks_shared(self, group_workspaces_mock):
```

Comment:
We test here if `_fit_shared_peak` is called correctly upon cal to `_fit_peaks`, can we add some tests to test the `_fit_shared_peak` function itself?

This should be easier to do once it has been refactored into several functions (each with tests ideally).